pandas/core/generic.py

```diff
@@ -1200,7 +1200,7 @@ def to_sql(self, name, con, flavor=None, schema=None, if_exists='fail',
                    if_exists=if_exists, index=index, index_label=index_label,
                    chunksize=chunksize, dtype=dtype)
 
-    def to_pickle(self, path):
+    def to_pickle(self, path, protocol):
```

Comment:
`protocol` should be an optional argument here. You'll probably need to have the default be `None`, and then determine the correct protocol based on the python version (see `pandas.compat`). The default should be the same as python's default.